wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -337,24 +338,27 @@ private void renderCombinedEventScripts()
 		if (combinedScript.length() > 0)
 		{
 			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_HANDLERS_BOUND);");
-			getRealResponse().render(
-				OnDomReadyHeaderItem.forScript(combinedScript.append('\n').toString()));
+			getRealResponse().render(OnDomReadyHeaderItem.forScript(combinedScript.append('\n')));
 		}
 
 		combinedScript.setLength(0);
+		String lastId = null;
```

Comment:
Would appreciate review on this commit, especially this part ....

Reply:
That doesn't seem to be reliable: what if multiple items are given an id?

IMHO we shouldn't changed code just to please test code. Couldn't BaseWicketTester just parse the url from the whole JavaScript via a regular expression?

> `.*Wicket\.Ajax\.baseUrl="(.*)";.*`